task.tf

```diff
@@ -38,17 +38,6 @@ resource "aws_ecs_task_definition" "fargate" {
     content {
       host_path = lookup(volume.value, "host_path", null)
       name      = volume.value.name
-
```

Comment:
I don't think it is necessary to remove this code. You just need type `any` instead of `map(string)` for the volume.

Reply:
Code doesn't serve any purpose right now, so we can remove it until such time as it works under Fargate.